### PR TITLE
Make the SessionIDEditable component a textarea rather than a co…

### PR DESCRIPTION
**Pull request by Bilb**
editable divs create lots of issues such as being able to paste html code.

### CODE REVIEW

File: ts/components/session/LeftPaneChannelSection.tsx [hunk 1]
```diff
@@ -297,11 +295,8 @@ export class LeftPaneChannelSection extends React.Component<Props, State> {
     );
   }
 
-  private handleOnPasteUrl(e: any) {
-    if (e.target.innerHTML) {
-      const cleanText = e.target.innerHTML.replace(/<\/?[^>]+(>|$)/g, '');
```
Comment (neuroscr) [inline ts/components/session/LeftPaneChannelSection.tsx]: do we have something else where that strip outs tags? or that not a concern?

Comment (Bilb) [reply ts/components/session/LeftPaneChannelSection.tsx]: It's a textarea now, so you can type html if you want but it won't be rendered as it, just as plain text.

Comment (neuroscr) [reply ts/components/session/LeftPaneChannelSection.tsx]: Just wanted to make sure that was on the receiving end as well, so we don't have an XSS here.

Comment (Bilb) [reply ts/components/session/LeftPaneChannelSection.tsx]: What we do with this string is test it against this regex:
`const regexURL = /(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?/;
`

and then attempt a connection with that url.

**Pull request merged**